app/src/org/commcare/dalvik/activities/CommCareHomeActivity.java

```diff
@@ -1050,37 +1064,51 @@ public boolean onPrepareOptionsMenu(Menu menu) {
     public boolean onOptionsItemSelected(MenuItem item) {
         switch (item.getItemId()) {
             case MENU_PREFERENCES:
+                reportOptionsItemEntry(GoogleAnalyticsFields.LABEL_SETTINGS);
```

Comment:
Oof, this is pretty ugly. I think making an item ID -> Key map and adding the event against that map above the switch might be a cleaner way to implement this.